### PR TITLE
fix linter issues, removed uneeded files from directory and .gitignore

### PR DESCRIPTION
**Pull request by Khaled-AlKhateeb**
please review the new modified branch that fixed issues was suggested, thank you

### CODE REVIEW

File: html-css/.github/workflows/linters.yml [hunk 1]
```diff
@@ -53,4 +53,4 @@ jobs:
       - uses: actions/checkout@v2
```
Comment (topeogunleye) [inline html-css/.github/workflows/linters.yml:53]: - [x] I kindly suggest that you move your `.github/workflows` folder out of your `html-css` folder and put them in your root directory. 
 The linters needed for this project can be found [here](https://github.com/microverseinc/linters-config/tree/master/html-css);
Here is a video demo I created demostrating  the above:

https://user-images.githubusercontent.com/52934128/164221056-46f41b39-cb1e-4fd9-a7f9-dd47bf835439.mp4
- [x] Kindly ensure your linter results are green * AKA: Linter should report NO errors.

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @Khaled-AlKhateeb, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
* Good commit messages. :heavy_check_mark:
* No linter errors. :heavy_check_mark:
* Followed Github flow :heavy_check_mark:

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

You can also consider:
>- [x] I suggest you add all headers from the Microverse [README template](https://github.com/microverseinc/readme-template) as this will give users detailed information about your project. Here is demo below

https://user-images.githubusercontent.com/52934128/176309876-98f49c4e-3e62-4edd-9bf3-51a66f48e2cd.mp4
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Khaled-AlKhateeb** [comment]
Please review my code again after fixing and changing.

File: html-css/.hintrc [hunk 2]
```diff
@@ -15,4 +15,4 @@
     "meta-viewport",
     "no-inline-styles:error"
   ]
-}
+}
```
Comment (1GORDON) [inline html-css/.hintrc]: Great job @Khaled-AlKhateeb 🤝

- [ ] As suggested by the previous reviewer, kindly delete the two folders of `html-css` and `javascript`. Then add the `.hintrc` file and the `.stylelintrc.json` file to the rrot folder.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Khaled-AlKhateeb ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Khaled-AlKhateeb** [comment]
kindly check my progress I hope I did everything right, thanks Gordon

**Review by sumairq — APPROVED** [review]
### Hi @Khaled-AlKhateeb  , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**